src/frontend/Komponenter/Journalforing/JournalforingAdmin.tsx

```diff
@@ -116,60 +101,111 @@ export const JournalforingAdmin: React.FC = () => {
+                            <Normaltekst>
+                                JournalpostID: {journalResponse.journalpost.journalpostId}
+                            </Normaltekst>
+                            <Normaltekst>
+                                Status:{' '}
+                                {journalstatusTilTekst[journalResponse.journalpost.journalstatus]}
+                            </Normaltekst>
+                            <Normaltekst>
+                                Dato mottatt:{' '}
+                                {formaterIsoDatoTid(journalResponse.journalpost.datoMottatt)}
+                            </Normaltekst>
+                        </Blokk>
+                        <Blokk>
+                            <Systemtittel>Behandlingstype</Systemtittel>
+                            {nyBehandlingstype && (
+                                <Normaltekst>
+                                    {behandlingstypeTilTekst[nyBehandlingstype]}
+                                </Normaltekst>
+                            )}
+                        </Blokk>
+
+                        <Hovedknapp
+                            onClick={() => sendInn(journalResponse, fagsak.id)}
+                            spinner={senderInn}
+                        >
+                            Opprett behandling
+                        </Hovedknapp>
+                        {!journalResponse.harStrukturertSøknad && (
+                            <AlertStripeAdvarsel>
+                                Kan ikke finne en digital søknad på denne journalposten.
+                            </AlertStripeAdvarsel>
                         )}
-                    </Blokk>
+                        {feilmelding && <AlertStripeFeil>{feilmelding}</AlertStripeFeil>}
+                    </SideLayout>
+                )}
+            </DataViewer>
+        </>
+    );
+};
+
+const UtledEllerVelgFagsak: React.FC<{
+    journalResponse: IJojurnalpostResponse;
+    hentFagsak: (personIdent: string, stønadstype: Stønadstype) => void;
+}> = ({ journalResponse, hentFagsak }) => {
+    {
+        const stønadstypeFraJournalpost = behandlingstemaTilStønadstype(
+            journalResponse.journalpost.behandlingstema
+        );
+        const [stønadstype, settStønadstype] = useState(stønadstypeFraJournalpost);
+
+        useEffect(() => {
+            if (stønadstype) {
+                hentFagsak(journalResponse.personIdent, stønadstype);
+            }
+            // eslint-disable-next-line
+        }, [stønadstype]);
 
-                    <Hovedknapp
-                        onClick={() => sendInn(journalResponse, fagsak.id)}
-                        spinner={senderInn}
+        if (!stønadstypeFraJournalpost) {
+            return (
+                <SideLayout>
+                    <AlertStripeInfo>
+                        Journalposten har ikke behandlingstema. Velg stønadstype.
+                    </AlertStripeInfo>
+                    <Select
```

Comment:
Burde vi vise tittel fra dokumentene her? (For å få litt ekstra info til valget av stønadstype)

Reply:
Det kunne kanskje vært greit - iallefall for "normal" journalføring..?

Reply:
Enig, tar det i neste PR 👍